src/javaRosa.js

```diff
@@ -1606,12 +1609,14 @@ define([
                         for (var k = 0; k < forms.length; k++) {
                             form = forms[k];
                             val = form.getValueOrDefault(lang);
-                            xmlWriter.writeStartElement("value");
-                            if(form.name !== "default") {
-                                xmlWriter.writeAttributeString('form', form.name);
+                            if (val) {
```

Comment:
Yeah. The test that I changed here was the same test I added in that PR. The only time it should evaluate to false is when it is `""`. I haven't seen a case where an itext form would be false, undefined, or null. Cases where a form shouldn't be written are handled by removing it from the list of forms.